utils/directories.cc

```diff
@@ -115,6 +119,10 @@ future<> directories::do_verify_owner_and_mode(fs::path path, recursive recurse,
     }
     switch (sd.type) {
     case directory_entry_type::regular: {
+        if (!filterFunc(path)) {
```

Comment:
nit: coding style: we prefer camel_case over camelCase (and try not mix dufferent coding styles in the same unit)

Reply:
Also, I think it would better to give it a more descriptive name than filter_func.  Maybe do_verify, so the condition will be clearer to the reader.

Reply:
Done.